### DISCUSSION TITLE
I would like to know how to modify this " A Tauri App", or where I can configure it.

### DISCUSSION THREAD

**Discussion by yourenz** [tauri-apps/tauri#8568] [Q&A] [answered]
This is my repo.: https://github.com/yourenz/pure-music

![image](https://github.com/tauri-apps/tauri/assets/31525716/15e2ef6d-7c1b-4474-8db3-fbea439db196)
![image](https://github.com/tauri-apps/tauri/assets/31525716/7c0f34dc-bc42-4ec7-9a35-3f38dde4c9ca)

**Comment by FabianLars** [comment]
It should be the [`shortDescription`](https://tauri.app/v1/api/config/#bundleconfig.shortdescription) in tauri.conf.json. If it's not set it will fall back to the `description` property in Cargo.toml, which is set to `"A Tauri App"` by default.

**Reply by yourenz** [reply]
Thanks for your answer, I did try to set **shortDescription** and **longDescription** and **description**  , but it didn't work.

**Reply by FabianLars** [reply, marked as answer]
Okay, i checked it out myself and it is a bit confusing.
in `tauri dev` it uses the description but you have to delete a registry key for it to update after changing it (look for the app's target path in a subfolder of `Computer\HKEY_CLASSES_ROOT\Local Settings\Software\Microsoft\Windows\Shell\MuiCache`).
in `tauri build` it seems to use the app/product name, but this requires the app to be installed via the msi or setup.exe installers first.

**Reply by yourenz** [reply]
I deleted the application data under the `computer\HKEY_CLASSES_ROOT\Local Settings\Software\Microsoft\Windows\Shell\MuiCache `directory.

Run `tauri dev` again, it's back to normal, it uses the tauri.conf.json package productName.

But there is still a problem, in `tauri dev`, modified tauri.conf.json package productName , it is still the previous name, not the latest change, only kill the current process, run `tauri dev` again, it will become the latest name.

The `tauri build` exe also uses the tauri.conf.json package productName .

This problem has been bothering me for a long time, thanks again for your help.
